src/jsontestrunner/main.cpp

```diff
@@ -56,7 +56,7 @@ static Json::String readInputTestFile(const char* path) {
     return "";
   fseek(file, 0, SEEK_END);
   long const size = ftell(file);
-  size_t const usize = static_cast<unsigned long>(size);
+  auto const usize = static_cast<unsigned long>(size);
```

Comment:
I think clang-tidy messed up here. We had a static_cast to unsigned long captured by a variable declared as const size_t. The conversion to auto is only a no op when unsigned long is the same type as a size_t.
Really the better change here would probably be to `const auto usize = static_cast<size_t>(size);`

Reply:
Agreed.

Reply:
Yeah, I definitely don't like the way the code was previously.